core/Metrics/Formatter.php

```diff
@@ -72,10 +72,14 @@ public function getPrettyTimeFromSeconds($numberOfSeconds, $displayTimeAsSentenc
 
         // Display 01:45:17 time format
         if ($displayTimeAsSentence === false) {
-            $hours   = floor($numberOfSeconds / 3600);
-            $minutes = floor(($reminder = ($numberOfSeconds - $hours * 3600)) / 60);
+            $days    = floor($numberOfSeconds / 86400);
+            $hours   = floor(($reminder = ($numberOfSeconds - $days * 86400)) / 3600);
+            $minutes = floor(($reminder = ($reminder - $hours * 3600)) / 60);
             $seconds = floor($reminder - $minutes * 60);
-            $time    = sprintf("%02s", $hours) . ':' . sprintf("%02s", $minutes) . ':' . sprintf("%02s", $seconds);
+            if($days == 0)
+                $time    = sprintf("%02s", $hours) . ':' . sprintf("%02s", $minutes) . ':' . sprintf("%02s", $seconds);
+            else    
+                $time    = sprintf("%02s days, ", $days) . sprintf("%02s", $hours) . ':' . sprintf("%02s", $minutes) . ':' . sprintf("%02s", $seconds);
```

Comment:
"days" needs to be translated. You can see it a few lines further below how it works. You want to use the translation key `Intl_NDays`. Eg like `Piwik::translate('Intl_NDays', $days)`